README.md

```diff
@@ -429,6 +429,7 @@ All endpoints are JSON.
 
  - **GET /proxies** - List existing proxies and their toxics
  - **POST /proxies** - Create a new proxy
+ - **POST /populate** - Create or replace a list of proxies
```

Comment:
I think we need some more documentation on this endpoint, which parameter it takes, etc.